test/tests/u-options.json

```diff
@@ -120,7 +120,7 @@
     {
       "locale": "ar",
       "src": "أهلاً {بالعالم :string}",
-      "exp": "أهلاً \u2067بالعالم\u2069"
+      "exp": "أهلاً \u2068بالعالم\u2069"
     }
```

Comment:
We may need to drop this test, or more explicitly define whether `:string` applies introspection to its value to determine its directionality. As we don't forbid introspection, an implementation could use either an RLI or FSI here.

Presuming that the introspection would follow UAX#9, the user's perception of the results would be the same, so my preference would be to make the character-level output predictable by requiring that no introspection is done in the absence of a `u:dir` override or an operand that includes an explicit direction.